tests/unit/s2n_session_ticket_test.c

```diff
@@ -465,7 +473,11 @@ int main(int argc, char **argv)
         /* Set session state lifetime for 15 hours which is equal to the default lifetime of a ticket key */
         EXPECT_SUCCESS(s2n_config_set_session_state_lifetime(server_config, S2N_SESSION_STATE_CONFIGURABLE_LIFETIME_IN_SECS));
 
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1), ticket_key1, s2n_array_len(ticket_key1), 0));
+        /* Add one session ticket key with an intro time in the past so that the key is immediately valid */
+        POSIX_GUARD(server_config->wall_clock(server_config->sys_clock_ctx, &now));
+        uint64_t key_intro_time = (now / ONE_SEC_IN_NANOS) - 1;
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1),
+                ticket_key1, s2n_array_len(ticket_key1), key_intro_time));
```

Comment:
This test is labeled:
>     /* Client sends non-empty ST extension. Server does a
>      * full handshake and issues a NST because the key is not found.
>      */

Is it still testing the "key is not found" use case? How was it testing that use case before your change?
Maybe the comment meant "ticket is not found" instead?

Reply:
The key originally used to encrypt the session ticket was not found. I can correct the comment.